zproject/urls.py

```diff
@@ -220,6 +218,12 @@
         {'PATCH': 'patch_bot_backend',
          'DELETE': 'deactivate_bot_backend'}),
 
+) + patterns('zerver.views.upload',
+    url(r'^user_uploads/(?P<realm_id_str>(\d*|unk))/(?P<filename>.*)', 'rest_dispatch',
+            {'GET': 'api_get_uploaded_file'}),
```

Comment:
Yeah, we'll want to make that an option that can be specified for just that one route, ideally something in the urls.py configuration...